mangaki/mangaki/utils/anidb.py

```diff
@@ -1,10 +1,13 @@
 from datetime import datetime
+from typing import Dict, List, Optional
 from urllib.parse import urljoin
 
 import requests
 from bs4 import BeautifulSoup
+from lazy_object_proxy.utils import cached_property
```

Comment:
What is `lazy_object_proxy`? Why not `from django.utils.functional import cached_property`?